ops/test/api.tf

```diff
@@ -7,6 +7,9 @@ module "simple_report_api" {
   resource_group_name     = data.azurerm_resource_group.rg.name
 
   docker_image_uri = "DOCKER|simplereportacr.azurecr.io/api/simple-report-api-build:${var.acr_image_tag}"
+
+  webapp_subnet_id = data.terraform_remote_state.persistent_test.outputs.subnet_webapp_id
```

Comment:
Does this need to be in other environments?

Reply:
Yes, this is the thing that needs to be added (including changes to the persistent module) to envs other than test and prod.